lib/auxiliary/multi_logger.rb

```diff
@@ -3,10 +3,10 @@
 # AutoHCK module
 module AutoHCK
   # MultiLogger class
-  class MultiLogger
+  class MultiLogger < Logger
     attr_reader :level
 
-    def initialize(*loggers)
+    def initialize(*loggers) # rubocop:disable Lint/MissingSuper
```

Comment:
Please move this change to separate commit

As a rule of thumb please always have cosmetics, unrelated to main topic changes, and etc as a separate commit